11:07:04 (2, <test_dailyRcmd.musicTest2 testMethod=test_1_intoDailyRcmd>, 'error pull\n', 'Traceback (most recent call last):\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 26, in wrapped\n    proxy.wait_for_appearance(timeout=proxy.poco._pre_action_wait_for_appearance)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 681, in wait_for_appearance\n    raise PocoTargetTimeout(\'appearance\', self)\npoco.exceptions.PocoTargetTimeout: Waiting timeout for appearance of "UIObjectProxy of "com.netease.cloudmusic:id/agree""\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/demo/testcase/music/test_dailyRcmd.py", line 26, in test_1_intoDailyRcmd\n    self.music_Test.mobilelogin("14010000028", "555555")\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/demo/common/music_Test.py", line 138, in mobilelogin\n    self.poco("com.netease.cloudmusic:id/agree").click()\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 29, in wrapped\n    raise e\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 23, in wrapped\n    return func(proxy, *args, **kwargs)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 332, in click\n    pos_in_percentage = self.get_position(focus)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 72, in wrapped\n    return func(proxy, *args, **kwargs)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 613, in get_position\n    pos = list(map(float, self.attr(\'pos\')))\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 39, in wrapped\n    return func(self, *args, **kwargs)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 736, in attr\n    nodes = self._do_query(multiple=False)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 890, in _do_query\n    raise PocoNoSuchNodeException(self)\npoco.exceptions.PocoNoSuchNodeException: Cannot find any visible node by query UIObjectProxy of "com.netease.cloudmusic:id/agree"\n')
11:07:16 (2, <test_dailyRcmd.musicTest2 testMethod=test_1_intoDailyRcmd>, 'error pull\n', 'Traceback (most recent call last):\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 26, in wrapped\n    proxy.wait_for_appearance(timeout=proxy.poco._pre_action_wait_for_appearance)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 681, in wait_for_appearance\n    raise PocoTargetTimeout(\'appearance\', self)\npoco.exceptions.PocoTargetTimeout: Waiting timeout for appearance of "UIObjectProxy of "com.netease.cloudmusic:id/agree""\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/demo/testcase/music/test_dailyRcmd.py", line 26, in test_1_intoDailyRcmd\n    self.music_Test.mobilelogin("14010000028", "555555")\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/demo/common/music_Test.py", line 138, in mobilelogin\n    self.poco("com.netease.cloudmusic:id/agree").click()\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 29, in wrapped\n    raise e\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 23, in wrapped\n    return func(proxy, *args, **kwargs)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 332, in click\n    pos_in_percentage = self.get_position(focus)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 72, in wrapped\n    return func(proxy, *args, **kwargs)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 613, in get_position\n    pos = list(map(float, self.attr(\'pos\')))\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 39, in wrapped\n    return func(self, *args, **kwargs)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 736, in attr\n    nodes = self._do_query(multiple=False)\n  File "/Users/chenjinli/PycharmProjects/TestObject/pythonProject/venv/lib/python3.9/site-packages/poco/proxy.py", line 890, in _do_query\n    raise PocoNoSuchNodeException(self)\npoco.exceptions.PocoNoSuchNodeException: Cannot find any visible node by query UIObjectProxy of "com.netease.cloudmusic:id/agree"\n')
